.eslintrc.js

```diff
@@ -8,21 +8,21 @@ module.exports = {
         node: true,
     },
     globals: {
-        'process': true,
         'monaco': true,
+        'process': true
     },
     // Ignore Lint Rules that are Overkill
     rules: {
         'no-arrow-condition': 'off',
         'space-before-blocks': 'off',
         'space-return-throw-case': 'off',
         'vue/attributes-order': 'off',
-        'vue/html-indent': 'off',
         'vue/html-closing-bracket-newline': 'off',
         'vue/html-closing-bracket-spacing': 'off',
+        'vue/html-indent': 'off',
         'vue/html-self-closing': 'off',
         'vue/max-attributes-per-line': 'off',
-        'vue/singleline-html-element-content-newline': 'off',
-        'vue/order-in-components': 'off'
+        'vue/order-in-components': 'off',
+        'vue/singleline-html-element-content-newline': 'off'
```

Comment:
@z1haze all I did here was sort these alphabetically, since I had not done that correctly before ( I like to keep things tidy ... most of the time, ha ha )